indico/modules/users/views.py

```diff
@@ -41,7 +44,11 @@ class WPUser(WPJinjaMixin, WPMainBase):
     template_prefix = 'users/'
 
     def _getNavigationDrawer(self):
-        return WSimpleNavigationDrawer(_('My Profile'))
+        if 'user_id' in request.view_args:
+            profile_breadcrumb = _('Profile of {0.full_name}').format(User.get(request.view_args['user_id']))
```

Comment:
IMHO that's nicer:

    user = User.get(request.view_args['user_id'])
    profile_breadcrumb = _('Profile of {name}').format(name=user.full_name)